lib/travis/build/script/r.rb

```diff
@@ -0,0 +1,368 @@
+module Travis
+  module Build
+    class Script
+      class R < Script
+        DEFAULTS = {
+          # Basic config options
+          cran: 'http://cran.rstudio.com',
+          warnings_are_errors: false,
+          # Dependencies (installed in this order)
+          apt_packages: [],
+          r_binary_packages: [],
+          r_packages: [],
+          bioc_packages: [],
+          r_github_packages: [],
+          # Build/test options
+          r_build_args: '--no-build-vignettes --no-manual',
```

Comment:
i had left latex optional, for time savings; i'm starting to think i should have it always install, in particular for things like this (since we really *do* want to check the vignettes and manual by default), and that's one more reason to get latex baked into the base image.